doc/en/installation/common/sql_strict_mode.rst

```diff
@@ -0,0 +1,13 @@
+.. warning::
+    Centreon started the compatibility with SQL strict mode but not all components are ready yet. It is mandatory to
+    disable the strict mode if you use MariaDB >= 10.2.4 or MySQL >= 5.7.5. In this case, connect to your DBMS with
+    root account and execute the following SQL request: ::
+        
+        # SET sql_mode = 'NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION';
+        # SET GLOBAL sql_mode = 'NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION';
+    
+    or modify the */etc/my.cnf.d/centreon.cnf* file to add in the '[server]' section the following line: ::
+        
+        sql_mode = 'NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION'
```

Comment:
does not work on deported mySql 8 DB

Reply:
are you sure ? cause it looks the same here : https://kevdees.com/disable-mysql-strict-mode-and-no_zero_date-errors-in-homestead/

Reply:
2019-10-10T11:38:56.347451Z 0 [ERROR] [MY-000077] [Server] /usr/sbin/mysqld: Error while setting value 'NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION' to 'sql_mode'.

Reply:
[root@mysql mysql]# mysql -V
mysql  Ver 8.0.17 for Linux on x86_64 (MySQL Community Server - GPL)

Reply:
indeed, "NO_AUTO_CREATE_USER" does not exist anymore since MySQL 8.0.11

Reply:
furthermore, we really need to split the documentation for mariaDB and MySQL, has this kind of behavior will be more frequent. And incoherences are already multiples in the current page (some command line are MySQL8 specific, other mariaDB)
@jlamotte has already began to check it

Reply:
In mysql 8 for disable strict mode SET GLOBAL sql_mode = 'NO_ENGINE_SUBSTITUTION'

Reply:
you right